accurest-core/src/test/groovy/io/codearte/accurest/dsl/WiremockGroovyDslSpec.groovy

```diff
@@ -863,6 +867,62 @@ class WiremockGroovyDslSpec extends WiremockSpec {
             e.message.contains "Query parameter 'age' can't be of a matching type: NOT_MATCHING for the server side"
 	}
 
+	def "should not allow query parameter with a different absent variation for server/client"() {
```

Comment:
This you could have written with the `where:` block where you would have the result of the `GroovyDsl.make` method. WDYT?